proxy/hdrs/HdrToken.cc

```diff
@@ -548,6 +548,30 @@ hdrtoken_tokenize_dfa(const char *string, int string_len, const char **wks_strin
   return wks_idx;
 }
 
+/*-------------------------------------------------------------------------
+  Have to work around that methods are case insensitive while the DFA is
+  case insensitive.
+  -------------------------------------------------------------------------*/
+
+int
+hdrtoken_method_tokenize(const char *string, int string_len)
```

Comment:
Note: performance perspective, it looks like this has some overhead, but it might not be matter. Because `GET` is processed in the fast path and doesn't come here.

https://github.com/apache/trafficserver/blob/f002cdbae85509f339f020051d68f8cecdf17bd1/proxy/hdrs/HTTP.cc#L935-L940